src/adapters/PeopleAdapter.js

```diff
@@ -0,0 +1,20 @@
+import {throwError} from 'rxjs';
+
+/**
+ * This is the base class that `PeopleAdapter` will inherit from.
```

Comment:
`This is a base class that defines the interface that maps people data.
Developers that want to extend `PeopleAdapater` must implement all of its methods, adhering to the exact parameters and structure of the returned objects
`